CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 2021.01.1
+
+**Released**: 2020.01.26
+
+**Commit Delta**: [Change from 2020.12.1 release](https://github.com/plus3it/spel/compare/2020.12.1...2021.01.1)
+
+**Manifests**: <https://github.com/plus3it/spel/blob/2021.01.1/manifests>
+
+**Summary**:
+
+*   "Extra" packages updated in this release:
+    - aws-cli/1.18.219
+    - aws-cli/2.1.21
+*   RHEL-only:
+    - rh-amazon-rhui-client-3.0.39
```

Comment:
@eemperor i wonder if this client version indicates this is RHUIv3?

Reply:
Yeah.  Might be.